framework/include/utils/TimedPrint.h

```diff
@@ -26,15 +26,17 @@
 #include <tuple>
 #endif
 
-#define CONSOLE_TIMED_PRINT(...)                                                                   \
+#define CONTROLLED_CONSOLE_TIMED_PRINT(initial_wait, dot_interval, ...)                            \
```

Comment:
Is this necessary? I think you can already pass in the initial and wait periods to the existing macro.